htdocs/compta/hrm.php

```diff
@@ -231,13 +231,11 @@
 			while ($i < $num && $i < $max)
 			{
 				$obj = $db->fetch_object($result);
-				$expensereportstatic->ref=$obj->rowid;
-				$expensereportstatic->id=$obj->rowid;
 				$userstatic->id=$obj->uid;
 				$userstatic->lastname=$obj->lastname;
 				$userstatic->firstname=$obj->firstname;
 				print '<tr '.$bc[$var].'>';
-				print '<td>'.$expensereportstatic->getNomUrl(1).'</td>';
+				print '<td><a href="../expensereport/card.php?id='.$obj->rowid.'">'.img_object($langs->trans("ShowTrip"),"trip").' '.$obj->ref.'</a></td>';
```

Comment:
Can you replace the <a href="../expensereport/card.php..... by <a href="'.DOL_URL_ROOT.'/expensereport/card.php..... ?